.github/workflows/pitest.yml

```diff
@@ -7,8 +7,40 @@ on:
   pull_request:
 
 jobs:
-  pitest-annotation:
+  pitest:
+    strategy:
+      matrix:
+        profile:
```

Comment:
`pitest-gui` ?

Reply:
```➜  checkstyle git:(idea-insp-1) ✗ cat pom.xml | grep "<id>pitest" | sort | cut -d ">" -f 2 | cut -d "<" -f 1
pitest-annotation
pitest-ant
pitest-api
pitest-blocks
pitest-coding-1
pitest-coding-2
pitest-coding-require-this-check
pitest-common-2
pitest-common
pitest-design
pitest-filters
pitest-gui
pitest-header
pitest-imports
pitest-indentation
pitest-java-ast-visitor
pitest-javadoc
pitest-main
pitest-metrics
pitest-misc
pitest-modifier
pitest-naming
pitest-packagenamesloader
pitest-regexp
pitest-sizes
pitest-tree-walker
pitest-utils
pitest-whitespace
pitest-xpath
```

Reply:
It has never been there in the build.

Reply:
Please add it, and leave it commented out with explanation that package needs better test coverage before adding to execution.

Reply:
Done.